zephyr/prj_qemu.conf

```diff
@@ -0,0 +1 @@
+CONFIG_TEST_RANDOM_GENERATOR=y
```

Comment:
If some specific configuration needs to be applied for qemu, you can add `boards/qemu_x86.conf` file to the sample, it'll be applied automatically for that platform.

Reply:
Removed the file